portal/templates/flask_user/flask_user_base.html

```diff
@@ -9,7 +9,7 @@
 {% block document_ready %}
 	{{super()}}
 	$(document).on("ready", function() {
-		$("input[type='text']").on("blur", function() {
+		$("input[type='text']").on("blur paste", function() {
 			$(this).val($.trim($(this).val()));
```

Comment:
What fields are being trimmed here? If it's user-entered fields it's probably fine but I worry that this is a behavior people will not expect when filling out a form.